nvm.sh

```diff
@@ -301,10 +301,13 @@ nvm_find_up() {
 
 
 nvm_find_nvmrc() {
-  local dir
-  dir="$(nvm_find_up '.nvmrc')"
-  if [ -e "${dir}/.nvmrc" ]; then
-    nvm_echo "${dir}/.nvmrc"
+  local legacy_nvm_dir
+  legacy_nvm_dir="$(nvm_find_up '.nvmrc')"
+  if [ -e "${legacy_nvm_dir}/.nvmrc" ]; then
+    nvm_echo "${legacy_nvm_dir}/.nvmrc"
+  fi
+  if [ -e "${XDG_CONFIG_HOME}/nvm/.nvmrc" ]; then
+    nvm_echo "${XDG_CONFIG_HOME}/nvm/.nvmrc"
```

Comment:
this will print out two lines if both locations exist.

I'm not actually sure why this is being changed; `.nvmrc` doesn't work currently inside `$NVM_DIR` when it's `$HOME/.nvm`; i'm not sure why it'd start working inside the XDG folder. If you want a global default, you'd do `nvm alias default whatever`.